src/libs/Pusher/pusher.js

```diff
@@ -214,15 +214,15 @@ function subscribe(
                 onResubscribe();
             });
 
-            channel.bind('pusher:subscription_error', (status) => {
-                if (status === 403) {
-                    Log.hmmm('[Pusher] Issue authenticating with Pusher during subscribe attempt.', {
-                        channelName,
-                        status,
-                    });
-                }
-
-                reject(status);
+            channel.bind('pusher:subscription_error', (data) => {
+                const {type, error, status} = data;
```

Comment:
The signature of this error event has changed. Previously we were throwing an error that could be caught by any caller of `Pusher.subscribe()`. This can be an `AuthError` and then would get thrown with a `reject(undefined)`. So, this change gives us a bit more information about what went wrong. I think for the most part this would be an `AuthError` (what would happen when the `callback()` in the custom authorizer returns an error).